scripts/rest_examples/http-client.env.json

```diff
@@ -6,7 +6,9 @@
     "prefix": "api/provisioning/v1",
     "identity": "eyJpZGVudGl0eSI6IHsidHlwZSI6ICJVc2VyIiwgImFjY291bnRfbnVtYmVyIjoiMTMiLCAiaW50ZXJuYWwiOnsib3JnX2lkIjoiMDAwMDEzIn19fQo=",
     "source_id": "1",
+    "pubkey_id": "1",
     "region": "us-east-1",
+    "launch_template_id": "",
     "reservation-get-id": "1"
   }
 }
```

Comment:
Generalized pubkey id too, that is useful as 1 is my key, others should use their own keys (and add them to the seed script).